integration/firestore/firebase_export.ts

```diff
@@ -55,5 +55,16 @@ const Timestamp = firebase.firestore.Timestamp;
 const GeoPoint = firebase.firestore.GeoPoint;
 const FieldValue = firebase.firestore.FieldValue;
 const Blob = firebase.firestore.Blob;
+const loadBundle = firebase.firestore.loadBundle;
+const namedQuery = firebase.firestore.namedQuery;
```

Comment:
You shouldn't need this since you should be able to access `Firestore.loadBundle` in the integration tests.